grakn-test-profiles/pom.xml

```diff
@@ -122,46 +120,5 @@
                 </dependency>
             </dependencies>
         </profile>
-
-        <profile>
-            <id>scaling</id>
```

Comment:
I added it ages ago literally for running ScalingTestIT, which I have now removed